spec/models/email_spec.rb

```diff
@@ -15,7 +15,7 @@
           pub_created_display: ['al-Muhandisīn, al-Jīzah : Dār Halā lil-Nashr wa-al-Tawzīʻ, 2016.',
                                 'المهندسين، الجيزة : دار هلا للنشر والتوزيع، 2016.'],
           format: ['Book'],
-          holdings_1display: '{"9034559":{"location":"ReCAP","library":"ReCAP","location_code":"rcppa",'\
+          holdings_1display: '{"9034559":{"location":"Remote","library":"ReCAP","location_code":"rcppa",'\
```

Comment:
It would be nice if you could update the Voyager id with an Alma id. In this case it would be: '9991533593506421' vs "9034559" . Alma is the new system we started using. You can get the 'holdings_1display' from https://catalog.princeton.edu/catalog/9991533593506421/raw if you search for 'holdings_1display'. The information that is different in the holdings_1display is that "location": "Remote Storage" and "location_code": "recap$pa".